docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,112 @@
+---
+title: High availability of Aiven for MySQL®
+sidebar_label: High availability
+---
+
+Aiven for MySQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available, and a summary is provided in the table below:
+
+| Plan         | High availability features                                      | Backup history | Uptime  |
+| ------------ | --------------------------------------------------------------- | -------------- | ------- |
+| **Hobbyist** | Single node with limited availability                           | 2 days         | 98.5 %  |
+| **Startup**  | Single node with limited availability                           | 2 days         | 99 %    |
+| **Business** | Two nodes (primary + standby: basic high availability)          | 14 days        | 99.9 %  |
+| **Premium**  | Three nodes (primary + 2 x standby: stronger high availability) | 30-day         | 99.99 % |
+
+## About primary and standby nodes
+
+Aiven's Business and Premium plans offer primary nodes and standby nodes. A standby
+service is useful for multiple reasons:
+
+-   Provides another physical copy of the data in case of hardware,
+    software, or network failures
+-   Typically reduces the data loss window in disaster scenarios
+-   Provides a quicker database time to restore with a controlled
+    failover in case of failures, as the standby is already installed,
+    running, and synchronised with the data
+-   Can be used for read-only queries to reduce the load on the primary
+    server
+
+## Failure handling
+
+### Minor failures
+
+Minor failures, such as service process crashes or temporary loss of
+network access, are handled automatically by Aiven in all plans without
+any major changes to the service deployment. The service automatically
+restores normal operation once the crashed process is restarted or when network access is
+restored.
+
+### Severe failures
```

Comment:
I would add RTO and RPO in this section (and probably in the minor section as well).  With standby, RTO is quick and RPO is close to zero loss (only loosing the unreplicated trx, and this could be a lossles if we implement semi-sync).  Wo standby, the RPO is lossing as much as 5 minutes of data (to be confirmed, we upload binlogs every 5 minutes) and the RTO is variable as it involves restoring a backup and applying binlogs which will depend on the db size and the write load (can be as much as 6 hours and even more in some cases).

Reply:
To me an extra section for RTO/RPO feels a bit of a duplication, as those are explained below.